packages/block-library/src/calendar/block.json

```diff
@@ -16,7 +16,15 @@
 		}
 	},
 	"supports": {
-		"align": true
+		"align": true,
+		"color": {
+			"link": true,
+			"__experimentalSkipSerialization": [ "text", "background" ],
```

Comment:
Maybe we could leverage `__experimentalSelector` within this support's config to help theme.json and global styles to target the correct elements?